diff --git a/dist/build/index.js b/dist/build/index.js
index ec4dd332d8f57f10fbb149f28ff9c11d617bd7dd..2fe9c8f9849d29760b411b6d7e7a980107754a88 100644
--- a/dist/build/index.js
+++ b/dist/build/index.js
@@ -715,7 +715,7 @@ async function build(dir, experimentalAnalyze = false, reactProductionProfiling
             // the logic from packages/next/src/shared/lib/router/utils/sorted-routes.ts
             // but is instead specifically focused on code that can be shared
             // eventually with the development code.
-            (0, _validateapppaths.validateAppPaths)(appPaths);
+            // (0, _validateapppaths.validateAppPaths)(appPaths);
             // Interception routes are modelled as beforeFiles rewrites
             rewrites.beforeFiles.push(...(0, _generateinterceptionroutesrewrites.generateInterceptionRoutesRewrites)(appPaths, config.basePath));
             _buildcontext.NextBuildContext.rewrites = rewrites;
